### ISSUE TITLE
[Feature] yarn install immutable log

### ISSUE THREAD

**Issue by zdm** [yarnpkg/berry#657]
- [ ] I'd be willing to implement this feature
- [ ] This feature can already be implemented through a plugin

**Describe the user story**

Currently it is not possible to know, which dependency is prevents to install dependencies in immutable mode. I should guess, what dependency I should upgrade.

**Describe the solution you'd like**

It would be useful show the list of dependencies, that prevents install command to execute in immutable mode, with it current version in lock file and new version. Eg:
```
yarn install --immutable
eslint@5.0.0 -> eslint@6.0.0 [failed]
```

**Comment by arcanis** [comment]
Seems ok to me. I think for a start printing a simple diff between old lockfile and new lockfile would be more than enough. We can use the [`diff`](https://github.com/kpdecker/jsdiff#readme) package for that.

**Comment by arcanis** [comment]
This has been implemented in #660:

![image](https://user-images.githubusercontent.com/1037931/71681187-4d134000-2d8c-11ea-8499-9076c02f563d.png)
